src/mmw/js/src/draw/views.js

```diff
@@ -118,8 +118,9 @@ var SelectAreaView = Marionette.ItemView.extend({
         // Go about the business of adding the ouline and UTFgrid layers.
         if (endpoint && tableId !== undefined) {
             var ol = new L.TileLayer(endpoint + '.png'),
-                grid = new L.UtfGrid(endpoint + '.grid.json?callback={cb}',
+                grid = new L.UtfGrid(endpoint + '.grid.json',
```

Comment:
Did this change cause any downstream issues? The selection still works, so I'm guessing not.